application/app/src/main/java/uk/nhs/digital/uec/api/service/impl/FuzzyServiceSearchService.java

```diff
@@ -107,24 +91,21 @@ public List<DosService> retrieveServicesByGeoLocation(
     List<DosService> nonPopulatedLatLongServices = new ArrayList<>();
 
     for (DosService dosService : filteredDosServices) {
-
-      if (Objects.isNull(dosService.getLocation()) || Objects.isNull(searchLatitude) || Objects.isNull(searchLongitude)) {
+      if (Objects.isNull(dosService.getGeoLocation()) || Objects.isNull(searchLatitude) || Objects.isNull(searchLongitude)) {
         nonPopulatedLatLongServices.add(dosService);
-      } else {
-        GeoPoint source =
-          new GeoPoint(Double.parseDouble(searchLatitude), Double.parseDouble(searchLongitude));
-        if (dosService.getLocation().getLon() == 0D && dosService.getLocation().getLat() == 0D) {
+      }
+      else {
+        if (dosService.getGeoLocation().getLon() == 0D && dosService.getGeoLocation().getLat() == 0D) {
           nonPopulatedLatLongServices.add(dosService);
         }
-        dosService.setDistance(locationService.distanceBetween(source, dosService.getLocation()));
       }
     }
 
-    this.populateServiceDistancesWithNorthingAndEastings(nonPopulatedLatLongServices, searchPostcode);
+    //this.populateServiceDistancesWithNorthingAndEastings(nonPopulatedLatLongServices, searchPostcode);
```

Comment:
Remove the commented code if we don't need it any more

Reply:
Sure , i will remove